app/src/main/java/nerd/tuxmobil/fahrplan/congress/repositories/AppRepository.kt

```diff
@@ -116,6 +116,28 @@ object AppRepository {
             .flowOn(executionContext.database)
     }
 
+    private val refreshChangedSessionsSignal = MutableSharedFlow<Unit>()
+
+    private fun refreshChangedSessions() {
+        logging.d(javaClass.simpleName, "Refreshing changed sessions ...")
+        val requestIdentifier = "refreshChangedSessions"
+        parentJobs[requestIdentifier] = databaseScope.launchNamed(requestIdentifier) {
+            refreshChangedSessionsSignal.emit(Unit)
+        }
+    }
```

Comment:
This is a receptive pattern here right? -> I wondering if it could be dried out.